clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -4199,7 +4199,7 @@ AST_MATCHER_P_OVERLOAD(QualType, references, internal::Matcher<Decl>,
 ///     cxxRecordDecl(hasName("Y")))))
 ///   matches `y.m()`, `x.m()` and (g()).m(), but not `x.g()`.
 /// cxxMemberCallExpr(on(callExpr()))
-///   does not match `(g()).m()`, because the parens are not ignored.
+///   does match `(g()).m()`, because the parens are ignored.
```

Comment:
```suggestion
///   only matches `(g()).m()` (the parens are ignored).
```
How about this, based on: https://godbolt.org/z/Tn46qaEdc